src/provider/provider_tracking.c

```diff
@@ -623,8 +734,8 @@ ipcOpenedCacheEvictionCallback(const ipc_opened_cache_key_t *key,
     // could allocate the memory at address `ptr` before a call to umfMemoryTrackerRemove
     // resulting in inconsistent state.
     if (value->mapped_base_ptr) {
-        umf_result_t ret =
-            umfMemoryTrackerRemove(p->hTracker, value->mapped_base_ptr);
+        umf_result_t ret = umfMemoryTrackerRemoveIpcSegment(
+            p->hTracker, value->mapped_base_ptr);
         if (ret != UMF_RESULT_SUCCESS) {
             // DO NOT return an error here, because the tracking provider
             // cannot change behaviour of the upstream provider.
```

Comment:
Please update this comment and the error message below

Reply:
I have updated the error message, but I did not get how you suggested changing the comment.